hangman.py

```diff
@@ -0,0 +1,330 @@
+    take a while to finish.
+    """
+    print("Loading word list from file...")
+    # inFile: file
+    inFile = open(WORDLIST_FILENAME, 'r')
+    # line: string
+    line = inFile.readline()
+    # wordlist: list of strings
+    wordlist = line.split()
+    print("  ", len(wordlist), "words loaded.")
+    return wordlist
+
+
+def choose_word(wordlist):
+    """
+    wordlist (list): list of words (strings)
+
+    Returns a word from wordlist at random
+    """
+    return random.choice(wordlist)
+
+
+wordlist = load_words()
+
+
+def is_word_guessed(secret_word, letters_guessed):
+    '''
+    secret_word: string, the word the user is guessing; assumes all letters are
+      lowercase
+    letters_guessed: list (of letters), which letters have been guessed so far;
+      assumes that all letters are lowercase
+    returns: boolean, True if all the letters of secret_word are in letters_guessed;
+      False otherwise
+    '''
+    letters_guessed = set(letters_guessed)
+    new_set = set(secret_word)
+    if letters_guessed | new_set == letters_guessed:
+    #This command combine two sets, if  whole new_set is in letters_guessed, this will give us set letters_guessed
+    #If whole new_set is not in letters_guessed, than it will give us set, which is bigger, than letters_guessed
+        return True
+    else:
+        return False
+
+
+def get_guessed_word(secret_word, letters_guessed):
+    '''
+    secret_word: string, the word the user is guessing
+    letters_guessed: list (of letters), which letters have been guessed so far
+    returns: string, comprised of letters, underscores (_), and spaces that represents
+      which letters in secret_word have been guessed so far.
+    '''
+    list_guessed_word = []
+    for symbol in secret_word:
+        if symbol in letters_guessed:
+            list_guessed_word.append(symbol)
+        else:
+            list_guessed_word.append('_ ')
+    #Part of function below is needed to delete space, if not guessed letter is in the end of the word
+    if list_guessed_word[-1] == '_ ':
+        list_guessed_word[-1] = '_'
+    return ''.join(list_guessed_word)
+
+
+def get_available_letters(letters_guessed):
+    '''
+    letters_guessed: list (of letters), which letters have been guessed so far
+    returns: string (of letters), comprised of letters that represents which letters have not
+      yet been guessed.
+    '''
+    list_of_avaible_letters = []
+    for letters in string.ascii_lowercase:
+        if letters not in letters_guessed:
+            list_of_avaible_letters .append(letters)
+    return ''.join(list_of_avaible_letters)
+
+
+def output_information(guesses_left, letters_guessed):
+    '''
+    Main information for the player
+    '''
+    print("You have", guesses_left, "guesses left")
+    get_available_letters(letters_guessed)
+    print("Available letters:", get_available_letters(letters_guessed))
+    guessed_letter = input("Please guess a letter: ")
```

Comment:
It looks strange that you perform `input` in function that named `output_information`

Reply:
done